apigw-rest-api-lambda-node/example-pattern.json

```diff
@@ -0,0 +1,59 @@
+{
+    "title": "Hello World AWS Lambda and Amazon API Gateway REST API",
+    "description": "Create a simple Lambda Function connected to a REST API.",
+    "language": "Node",
+    "level": "200",
+    "framework": "SAM",
+    "introBox": {
+      "headline": "How it works",
+      "text": [
+        "This sample project demonstrates how to trigger a simple Lambda function using a REST API.",
+        "This pattern deploys one Lambda Function, andn one API Gateway REST API."
+      ]
+    },
+    "gitHub": {
+      "template": {
+        "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/apig-rest-api-lambda-node",
+        "templateURL": "serverless-patterns/apig-rest-api-lambda-node",
+        "projectFolder": "apig-rest-api-lambda-node",
+        "templateFile": "template.yaml"
+      }
+    },
+    "resources": {
+      "bullets": [
+        {
+          "text": "Trigger Lambda with a REST API",
+          "link": "https://docs.aws.amazon.com/lambda/latest/dg/services-apigateway.html"
+        },
+        {
+          "text": "Amazon API Gateway REST API",
+          "link": "https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-rest-api.html"
+        }
+      ]
+    },
+    "deploy": {
+      "text": [
+        "sam deploy --guided"
+      ]
+    },
+    "testing": {
+      "text": [
+        "See the GitHub repo for detailed testing instructions."
+      ]
+    },
+    "cleanup": {
+      "text": [
+        "Delete the application: <code>sam delete</code>."
+      ]
+    },
+    "authors": [
```

Comment:
Update your bio info.

Reply:
I didn’t fill out the author section of the example-pattern.json since I didn’t write this pattern, I am just porting it to serverless land from the SAM CLI templates [repo](https://github.com/aws/aws-sam-cli-app-templates/tree/master/nodejs20.x/hello). Should I just remove this section?

Reply:
Added the author section!